charts/hono/templates/_helpers.tpl

```diff
@@ -637,6 +637,9 @@ The scope passed in is expected to be a dict with keys
 Adds port type declarations to a component's service spec.
 */}}
 {{- define "hono.serviceType" }}
+{{- if .Values.serviceType }}
+  type: {{ .Values.serviceType }}
+{{- else }}
```

Comment:
I guess we can turn this into an `elseif` as well, right?

Reply:
updated